Remove extra full stop from function header of `get_operator`

hansrajdas opened the pull request.

hansrajdas committed:
Remove extra full stop from function header of `get_operator`

brandtbucher commented on Python/ast.c:
```diff
@@ -958,7 +958,7 @@ PyAST_FromNode(const node *n, PyCompilerFlags *flags, const char *filename_str,
 }
 
-/* Return the AST repr. of the operator represented as syntax (|, ^, etc.)
+/* Return the AST repr of the operator represented as syntax (|, ^, etc.)
```
Pretty sure "repr." here is short for "representation". This function has nothing to do with Python's `repr()`/`__repr__()`...

hansrajdas replied:
- Agreed, repr expanded to representation
- Extra line between function header and function body also deleted

brandtbucher reviewed (changes requested):
Not sure about this one @hansrajdas. See below:

hansrajdas committed:
repr expamded to representation because this function has nothing to do
with repr()/__repr__()
- Deleted extra line between function header and function body
- Added a space at closing of function header
